lib/kitchen/driver/azurerm.rb

```diff
@@ -218,10 +218,15 @@ class Azurerm < Kitchen::Driver::Base
         false
       end
 
+      default_config(:zone) do |_config|
```

Comment:
Can we make sure that https://kitchen.ci/docs/drivers/azurerm/ gets updated to describe this new config option. Otherwise no one will ever know it's an option. The docs for this live here: https://github.com/test-kitchen/test-kitchen/blob/master/docs/content/docs/drivers/azurerm.md